lnrpc/rpc_utils.go

```diff
@@ -17,8 +17,28 @@ func RPCTransactionDetails(txns []*lnwallet.TransactionDetail) *TransactionDetai
 
 	for i, tx := range txns {
 		var destAddresses []string
-		for _, destAddress := range tx.DestAddresses {
-			destAddresses = append(destAddresses, destAddress.EncodeAddress())
+		// Re-package destination output information.
+		var outputDetails []*OutputDetail
+		for _, o := range tx.OutputDetails {
+			// Note: DestAddresses is deprecated but we keep populating it
```

Comment:
nit: lines too long here and in the next commit.

Reply:
Ah yes, I may have had the standard 4-whitespace settting on. I have changed it accordingly and tried to make it more readable instead of many breaks.